package.json

```diff
@@ -114,6 +115,7 @@
     "Jicheng Li <deemstone@gmail.com>",
     "Christoffer Hasselberg <stofolus@gmail.com>",
     "David Herges <dherges@users.noreply.github.com>",
-    "Greg Magolan <gmagolan@gmail.com>"
+    "Greg Magolan <gmagolan@gmail.com>",
+    "Livio Brunner <livio.brunner.lb1@gmail.com>"
```

Comment:
:-)

Reply:
Don't know whether I am allowed to add me to this list - just wanted to share my contact information in case you need to get back to me :)